src/routes/process-user-vocabulary/UserVocabularyProcessScreen.tsx

```diff
@@ -162,14 +270,27 @@ const UserVocabularyProcessScreen = ({ navigation }: UserVocabularyProcessScreen
         />
         <StyledHintText>{maxPictureUpload}</StyledHintText>
         {hasImageErrorMessage && <ContentError>{getLabels().userVocabulary.creation.imageErrorMessage}</ContentError>}
-
-        <AddAudioButton
-          onPress={() => null}
-          label={addAudio}
-          buttonTheme={BUTTONS_THEME.text}
-          iconLeft={MicrophoneCircleIcon}
-          iconSize={theme.spacingsPlain.xl}
-        />
+        {recordingPath ? (
+          <AudioContainer style={{ opacity: fadeAnim }}>
+            <>
+              <AudioText>{recordingPath.substring(recordingPath.lastIndexOf('/') + 1).toUpperCase()}</AudioText>
```

Comment:
So here we are using the `react-native-audio-recorder-player` for playing the sound file and in all other cases we are using `react-native-sound-player` for playing audio. I think it would be good to reuse the existing AudioPlayer.tsx component for playing an audio to have this consistent.

Reply:
alright then i think we should refactor the `vocabularyItem` to just `audio` passend